modules/ting/src/Search/TingSearchRequest.php

```diff
@@ -233,6 +233,44 @@ public function setTermsPrFacet($terms_per_facet) {
     return $this;
   }
 
+  /**
+   * The facets used for this search request.
+   *
+   * @see TingSearchRequest::setFacets() for more information.
+   *
+   * @return string[]
+   *   The facets.
+   */
+  public function getFacets() {
+    return $this->facets;
+  }
+
+  /**
+   * Sets the facets used for this search request.
+   *
+   * Note that the facets set here might be provider dependent. Not all
+   * providers have the same facets available and if facets used is not defined
+   * in TingSearchCommonFields this may break search request for a given
+   * provider. So this should be used with care or it might limit the usefulness
+   * of the module using it.
+   *
+   * Modules using non TingSearchCommonFields facets should use those defined by
+   * ding_facetbrowser or at least make the facets used configurable in the
+   * sites administration menus.
```

Comment:
I get the intention here but the text is a bit misleading. It suggests that if developers use `TingSearchCommonFields` then any mapping will be handled for them. That is currently not the case.

I suggest that these the preceeding two paragraphs are reduced to something like:

> Note that facets set will be provider dependent. Search providers are likely not have the same facets available and referencing the id of a facet for a specific provider will may break search request for another provider. So this should be used with care or it might limit the usefulness of the module using it.
> Modules specifying facets to retrieve should make the facets used configurable in the site administration.